### ISSUE TITLE
Ability to put equations on page via the CMS text editor 

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2882]
We're currently using images to put equations and calculations on the site, but unnecessary images of text violate WCAG. Ideally, we'd be able to add equations in the text editor. 

Once we do this we'll need to make changes to all the RV detailed examples.

**Comment by greensomes** [comment]
See [MathJax](https://www.npmjs.com/package/mathjax). It's possible that an editorjs plugin to apply the library can be implemented.

**Comment by cthomasONRR** [comment]
![image](https://github.com/user-attachments/assets/9c623ae7-b287-4ddf-b5e6-8a0dcabc7b48)
